src/java/org/apache/cassandra/auth/CassandraRoleManager.java

```diff
@@ -508,6 +507,23 @@ private static String escape(String name)
         return StringUtils.replace(name, "'", "''");
     }
 
+    /** Allows selective overriding of the consistency level for specific roles. */
+    protected static ConsistencyLevel consistencyForRoleWrite(String role)
+    {
+        if (role.equals(DEFAULT_SUPERUSER_NAME))
+            return DEFAULT_SUPERUSER_CONSISTENCY_LEVEL;
+        else
+            return CassandraAuthorizer.authWriteConsistencyLevel();
```

Comment:
nit: I feel that a ternary operator might be more readable. The `else` between 2 return statements is confusing me.
Same thing in the next method.